templates/OperationUrlSnippet.stencil

```diff
@@ -1,7 +1,7 @@
 // Construct URL
-let urlTemplate = "{{ op.request.uri }}{{ op.request.path }}"
-let pathParams = [
-    {% for param in op.uriParams %}
+let urlTemplate = "{{ op.request.path }}"
+let pathParams:[String: String] = [
```

Comment:
Is there a case where omitting this type causes problems? Even in the empty scenario, you provide two strings, so the compiler should be able to figure this out.

Reply:
Removed.